app/templates/components/events/view/export/download-zip.hbs

```diff
@@ -10,23 +10,28 @@
 <div class="ui container"></div>
 <div class="ui form">
   <div class="field">
-    {{ui-checkbox class='toggle' label=(t 'Image')}}
+    {{ui-checkbox class='toggle' label=(t 'Image') checked=data.image onChange=(action (mut data.image))}}
   </div>
   <div class="field">
-    {{ui-checkbox class='toggle' label=(t 'Video')}}
+    {{ui-checkbox class='toggle' label=(t 'Video') checked=data.video onChange=(action (mut data.video))}}
   </div>
   <div class="field">
-    {{ui-checkbox class='toggle' label=(t 'Audio')}}
+    {{ui-checkbox class='toggle' label=(t 'Audio') checked=data.audio onChange=(action (mut data.audio))}}
   </div>
   <div class="field">
-    {{ui-checkbox class='toggle' label=(t 'Document')}}
+    {{ui-checkbox class='toggle' label=(t 'Document') checked=data.document onChange=(action (mut data.document))}}
   </div>
   <div class="ui basic segment less left padding">
-    <button class="ui blue button">
+    <button class="ui blue button" {{action startGeneration}}>
       {{t 'Start'}}
     </button>
-    <button class="ui button">
+    <a href="{{eventDownloadUrl}}" class="ui button {{if isDownloadDisabled 'disabled' ''}}">
       {{t 'Download'}}
-    </button>
+    </a>
+  </div>
+  <div class="ui basic segment">
+    <div class="ui header">
+      {{t 'Status'}} : {{eventExportStatus}}
```

Comment:
@geekyd I think it would be better to indicate the status to the user because in some cases the export might directly fail. Overall, we can show the status returned by API.